h2o-r/tests/testdir_algos/stackedensemble/runit_stackedensemble_base_models_using_grid.R

```diff
@@ -0,0 +1,147 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../../scripts/h2o-r-test-setup.R")
+
+# PUBDEV-4534
+stackedensemble_accepts_grid_and_models_as_base_models.test <- function() {
+    # This test asserts that base models can be one of these:
+    # * list of models
+    # * GridSearch
+    # * list of GridSearches
+    # * list of Gridsearches and models
+
+    get_base_models <- function(ensemble) {
+        unlist(lapply(ensemble@parameters$base_models, function (base_model) base_model$name))
+    }
+
+    expect_equal_unordered <- function (lhs, rhs) {
+        expect_equal(sort(lhs), sort(rhs))
+    }
+
+    train <- h2o.uploadFile(locate("smalldata/testng/higgs_train_5k.csv"),
+                            destination_frame = "higgs_train_5k")
+
+    valid <- h2o.uploadFile(locate("smalldata/testng/higgs_test_5k.csv"),
+                            destination_frame = "higgs_test_5k")
+
```

Comment:
please avoid this dataset for basic tests, `higgs_5k` is not that small, and we don't want the tests taking too long. Besides, most of the tests you have to write are for ensembles and automl, which means that the test has to train multiple models...

I'll rather recommend `prostate/prostate.csv`, `junit/cars.csv`, `junit/australia.csv`, `junit/iris.csv`, `junit/weather.csv`... let's say datasets < 50kB, and keep the larger ones to test distributed code, running on multiple nodes.

Reply:
I see that the Python counterpart is using the higgs dataset too: looks like I forgot to change it last time I modified this file. Feel free to change the dataset there as well, thx.